view/frontend/requirejs-config.js

```diff
@@ -0,0 +1,7 @@
+const config = {
+    map: {
+        '*': {
+            getid: 'Salesfire_Salesfire/js/getid',
```

Comment:
I think we need to change this to something more unique, maybe sfgetid as getid feels like it's more likely to be used